package.json

```diff
@@ -32,24 +32,26 @@
     "hps": "local-ssl-proxy --cert ./dev/localhost.pem --key ./dev/localhost-key.pem"
   },
   "dependencies": {
+    "decanter": "^7.0.0-rc.2",
     "dotenv": "^16.0.3",
-    "gatsby": "^5.0.0",
-    "gatsby-plugin-manifest": "^5.0.0",
-    "gatsby-plugin-sharp": "^5.0.0",
-    "gatsby-source-filesystem": "^5.0.0",
+    "gatsby": "^5.7.0",
+    "gatsby-plugin-manifest": "^5.7.0",
+    "gatsby-plugin-sharp": "^5.7.0",
+    "gatsby-source-filesystem": "^5.7.0",
     "gatsby-source-storyblok": "^5.0.0",
-    "gatsby-transformer-sharp": "^5.0.0",
+    "gatsby-transformer-sharp": "^5.7.0",
     "netlify-plugin-vault-variables": "^1.0.7",
     "node-vault": "^0.9.22",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
-    "@types/node": "^18.11.18",
-    "@types/react": "^18.0.27",
-    "@types/react-dom": "^18.0.10",
+    "@types/node": "^18.14.1",
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
```

Comment:
I nuked the node_modules folder and did a fresh install. Updated Gatsby and the essential packages to latest versions